tests/unit/common.c

```diff
@@ -58,22 +58,22 @@ void set_up_connection(void)
 {
   drizzle_return_t driz_ret;
   const char *verbosity;
-  
+
   ASSERT_NULL_(con, "con opened twice?");
 
-  con= drizzle_create(getenv("MYSQL_SERVER"),
                       getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
                       getenv("MYSQL_USER"),
                       getenv("MYSQL_PASSWORD"),
                       getenv("MYSQL_SCHEMA"), 0);
+  con = drizzle_create(getenv("MYSQL_SERVER"),
```

Comment:
It seems like some of the changes are shown out of order/wrongly. It compiles and the location of `drizzle_create(getenv("MYSQL_SERVER")` is where it should be. 
I will try to see if it changes after force pushing and rebasing on latest v5.4.x